pages/price-feeds/best-practices.mdx

```diff
@@ -70,6 +73,18 @@ This analogy suggests two simple solutions to races:
    The protocol can require a short delay between the two transactions, and the user's request gets fulfilled at the Pyth price as of the second transaction.
    This technique gives the protocol extra time to observe price changes, giving it a head start in the latency race.
 
+### Latency Mitigations for Derivative Protocols
```

Comment:
why derivatives in particular? I think these tips are applicable to any protocol right?

Reply:
I believe derivates are getting impacted by this issue the most.